### ISSUE TITLE
Validate fields when user presses continue/submit

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-refund#130]
Validate fields when the user clicks continue/submit; rather than as they go through the page

# If validation fails:
- User is returned to top of the page (Must have)
- Static error summary box added under the section title (Must have)
- Dynamic error summary box text explaining the problem and linking to the field (Should have)
- Add "Error:" before page name in <title>; but remove "Error:" when the user goes to the next stage

# Static error summary text:
## There is a problem
Please check your answers again

# Dynamic error summary text:
## There is a problem
<validation error text, with link to that field>

**Comment by mrmjprice** [comment]
<img width="1080" alt="Screenshot 2020-04-15 at 11 59 14" src="https://user-images.githubusercontent.com/10854265/79329952-8ffabd00-7f10-11ea-801f-fb33aeeed8c1.png">

**Comment by mrmjprice** [comment]
Error message component: https://wmnetwork.netlify.com/components/message/

**Comment by daylesalmon** [comment]
So the first error message has been implemented via PR #133
